packages/nuejs.org/docs/reactive-components.md

```diff
@@ -61,7 +61,7 @@ Inline handlers are good for simple expressions.
 
 
 ### Method handlers
-More complex functionality should go to an instance method
+More complex functionality should be relegated to an instance method:
```

Comment:
I think I wouldn't understand this on a first read.
Maybe `… should be moved to …`
What do you think?

Reply:
Relegated is a big word and I couldn't find a better one. 
"should be moved" is the way to go here.
Thanks.

Reply:
```suggestion
More complex functionality should be moved to an instance method:
```
Then I'll use that. Thanks for your review.